query/src/main/scala/filodb/query/exec/PromQlRemoteExec.scala

```diff
@@ -43,13 +44,9 @@ trait RemoteExec extends LeafExecPlan{
       .tag("query-id", queryContext.queryId)
       .start()
 
-    val queryResponse = PromQlExec.httpGet(params, queryContext.submitTime).
-      map { response =>
-        response.unsafeBody match {
-          case Left(error) => QueryError(queryContext.queryId, error.error)
-          case Right(successResponse) => toQueryResponse(successResponse.data, queryContext.queryId, execPlan2Span)
-        }
-      }
+    val httpEndpoint = params.remoteQueryEndpoint.get + params.remoteQueryPath.getOrElse("")
```

Comment:
Should remoteQueryEndpoint be part of RemoteExec and not be an Option? Since RemoteExec should also have remote query end point. If not we should throw an exception with some meanigful message if params.remoteQueryEndpoint is None. It will make debugging easier

Reply:
this is fixed